### PR TITLE
feat(venv): display project name and venv path instead in venv list

### PR DESCRIPTION
**Pull request by pzhlkj6612**
Screenshot:

Comparison of the venv view

## Summary by Sourcery

Update the virtual environment list view to show project names and relative venv paths, and add tests for the new display formatting.

New Features:
- Display the project name and venv path in the venv list instead of raw project paths and sysroot information.

Enhancements:
- Simplify the venv table layout by reordering and renaming columns to focus on project and venv path information.

Tests:
- Add unit tests for VenvListViewModel display formatting of project names and relative venv paths, including edge cases.

### CODE REVIEW

File: tests/org.ruyisdk.ruyi.tests/src/org/ruyisdk/venv/viewmodel/VenvListViewModelDisplayTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import org.junit.Test;
+import org.ruyisdk.venv.model.Venv;
+
+/**
+ * Unit tests for venv table display formatting.
+ */
+public class VenvListViewModelDisplayTest {
```
[the remark this answers is not in the document]
Comment (pzhlkj6612) [reply tests/org.ruyisdk.ruyi.tests/src/org/ruyisdk/venv/viewmodel/VenvListViewModelDisplayTest.java:12]: done, d390352d9ef7639c109ec13c0992edd3ad88ad37
